From 39ff45bfc1959049ad968c6ccf1624ec66b865ce Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Tue, 22 Jan 2013 18:03:22 +0800
Subject: [PATCH 154/310] ENGR00241208 Enlarge the navigation bar height

Enlarge the navigation bar height size to 55dp.
As we have set the dpi to 140, framework will
decrease the height size by 55dpx140/160 which
is actually 48px. Then the total usable window size
of each application is 1024x720. Under this
configuration, the one pixel highlight line below
the statusbar has gone.
This should be a bug in phone window policy when
Window manager decide which region can be used by
apps when status bar visiable.

We just put a workaround here.

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 .../frameworks/base/core/res/res/values/dimens.xml | 24 ++++++++++++++++++++++
 .../frameworks/base/core/res/res/values/dimens.xml | 24 ++++++++++++++++++++++
 2 files changed, 48 insertions(+)
 create mode 100644 sabreauto_6q/overlay/frameworks/base/core/res/res/values/dimens.xml
 create mode 100644 sabresd_6dq/overlay/frameworks/base/core/res/res/values/dimens.xml

diff --git a/sabreauto_6q/overlay/frameworks/base/core/res/res/values/dimens.xml b/sabreauto_6q/overlay/frameworks/base/core/res/res/values/dimens.xml
new file mode 100644
index 0000000..7532ad1
--- /dev/null
+++ b/sabreauto_6q/overlay/frameworks/base/core/res/res/values/dimens.xml
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+/* //device/apps/common/assets/res/any/dimens.xml
+**
+** Copyright 2006, The Android Open Source Project
+**
+** Licensed under the Apache License, Version 2.0 (the "License");
+** you may not use this file except in compliance with the License.
+** You may obtain a copy of the License at
+**
+**     http://www.apache.org/licenses/LICENSE-2.0
+**
+** Unless required by applicable law or agreed to in writing, software
+** distributed under the License is distributed on an "AS IS" BASIS,
+** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+** See the License for the specific language governing permissions and
+** limitations under the License.
+*/
+-->
+<resources>
+    <dimen name="navigation_bar_height">55dp</dimen>
+    <!-- Height of the bottom navigation bar in portrait; often the same as @dimen/navigation_bar_height -->
+    <dimen name="navigation_bar_height_landscape">55dp</dimen>
+</resources>
diff --git a/sabresd_6dq/overlay/frameworks/base/core/res/res/values/dimens.xml b/sabresd_6dq/overlay/frameworks/base/core/res/res/values/dimens.xml
new file mode 100644
index 0000000..7532ad1
--- /dev/null
+++ b/sabresd_6dq/overlay/frameworks/base/core/res/res/values/dimens.xml
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+/* //device/apps/common/assets/res/any/dimens.xml
+**
+** Copyright 2006, The Android Open Source Project
+**
+** Licensed under the Apache License, Version 2.0 (the "License");
+** you may not use this file except in compliance with the License.
+** You may obtain a copy of the License at
+**
+**     http://www.apache.org/licenses/LICENSE-2.0
+**
+** Unless required by applicable law or agreed to in writing, software
+** distributed under the License is distributed on an "AS IS" BASIS,
+** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+** See the License for the specific language governing permissions and
+** limitations under the License.
+*/
+-->
+<resources>
+    <dimen name="navigation_bar_height">55dp</dimen>
+    <!-- Height of the bottom navigation bar in portrait; often the same as @dimen/navigation_bar_height -->
+    <dimen name="navigation_bar_height_landscape">55dp</dimen>
+</resources>
-- 
1.8.0

